Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/LoginFragment.java

```diff
@@ -30,10 +27,37 @@
 
     FirebaseAuth auth;
```

Comment:
To avoid using Firebase stuff anywhere except the Service classes, what do you think about instantiating FirebaseAuth in the constructor for UserService (similar to what is done in MoodEventService) The methods couldn't be static in this case so you'd have to have a UserService object instantiated in LoginFragment but i think this is worth it to abstract the Firebase stuff